databricks/koalas/frame.py

```diff
@@ -2017,6 +2017,9 @@ def apply_batch(self, func):
         Apply a function that takes pandas DataFrame and outputs pandas DataFrame. The pandas
         DataFrame given to the function is of a batch used internally.
 
+        See also `Transform and apply a function
+        <https://koalas.readthedocs.io/en/stable/user_guide/transform_apply.html>`_.
```

Comment:
I'm wondering which we should use, stable or latest?

Reply:
Let me just change it to latest, thanks!